src/app/client/src/app/modules/groups/services/groups/groups.service.ts

```diff
@@ -250,4 +249,27 @@ getActivity(groupId, activity, mergeGroup) {
   getSupportedActivityList() {
     return this.groupCservice.getSupportedActivities();
   }
+
+  getActivityList (showList, groupData) {
```

Comment:
Why are we re-writting this logic. We should re-use the mobile logic here. 
Check with Anil or Ahmed

Reply:
This should be same for mobile & portal. It should behave the same.